analyzers/tests/SonarAnalyzer.Test/Rules/PasswordsShouldBeStoredCorrectlyTest.cs

```diff
@@ -36,21 +36,51 @@ public class PasswordsShouldBeStoredCorrectlyTest
             .AddReferences([
                 AspNetCoreMetadataReference.MicrosoftExtensionsIdentityCore,
                 AspNetCoreMetadataReference.MicrosoftAspNetCoreCryptographyKeyDerivation,
-                CoreMetadataReference.SystemSecurityCryptography,
+                ..MetadataReferenceFacade.SystemSecurityCryptography,
```

Comment:
Even though this test is `#NET` only, I think it's good practise to always use the Facade.